text/0000-glimmer-components.md

```diff
@@ -628,50 +678,93 @@ helpers:
 <div {{on-render setScrollPosition @scrollPosition}}></div>
 ```
 
-This RFC proposes adding two element modifiers:
+This RFC proposes adding two element modifiers: `{{on-render}}` and
+`{{on-destroy}}`.
+
+#### `{{on-render}}`
+
+This modifier runs immediately after the element is inserted, and again whenever
+any of the arguments passed to it update. It has the following timing semantics:
```

Comment:
Do we consider functions stable? Does `{{on-render (some-helper)}}` get called again (after initial render) if `some-helper` recomputes?

Reply:
yup, it will. Good to call that out specifically probably, but if _any_ of the arguments changes at all, it will rerun.